matter/si91x/siwx917/BRD4338A/autogen/sl_event_handler.c

```diff
@@ -14,7 +14,6 @@ void sl_platform_init(void) {
   sl_device_init_nvic();
   sli_si91x_platform_init();
   RSI_Board_Init();
-  sl_si91x_hardware_setup();

```

Comment:
What is the relation between sleep, lCD and this seemingly general hardware init? How will this be handled in code?

Reply:
same question as @jmartinez-silabs

Reply:
sl_si91x_hardware_setup();  this function disabling some modules which is required by LCD and because of this LCD init is failing.

We will take care of this when we enable sleep for common flash.
Anyway LCD and sleep are going to be enabled simultaneously